pkg/utils/test/testf/testf.go

```diff
@@ -106,6 +97,15 @@ func NewTestContext(opts ...TestContextOpt) (*TestContext, error) {
 		}
 	}
 
+	if tc.cfg == nil && tc.client == nil {
```

Comment:
You mean, tc.cfg is not need if there is client already?

Reply:
correct, I've split the option from the actual struct so it is not set if not needed